src/igvfd/searches/configs/primary_cell.py

```diff
@@ -42,6 +42,9 @@ def primary_cell():
             },
```

Comment:
L19: donors.taxa should be removed (have 'taxa' listed already)

Reply:
fixed